app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/statistikk/StatistikkModel.kt

```diff
@@ -296,20 +314,53 @@ class StatistikkModel(
                 }
             }
             is HardDelete -> {
-                // noop
+                if (hendelse.grupperingsid != null && hendelse.merkelapp != null) {
+                    database.nonTransactionalExecuteUpdate(
+                        """
+                        update notifikasjon 
+                            set hard_deleted_tidspunkt = ?
+                            where gruppering_id = ? and merkelapp = ?
+                        """
+                    ) {
+                        timestamp_without_timezone_utc(hendelse.deletedAt)
+                        text(hendelse.grupperingsid)
+                        text(hendelse.merkelapp)
+                    }
+                }
+                database.nonTransactionalExecuteUpdate(
+                    """
+                    update notifikasjon 
+                        set hard_deleted_tidspunkt = ?
+                        where notifikasjon_id = ?
+                    """
+                ) {
+                    timestamp_without_timezone_utc(hendelse.deletedAt)
+                    uuid(hendelse.aggregateId)
+                }
+                database.nonTransactionalExecuteUpdate(
+                    """
+                    update sak 
```

Comment:
Med denne endringen trenger vi ikke en egen mikrotjeneste for å sjekke om ting opprettes etter delete, men vi kan bygge det inn i Statistikk appen. Som jo gir mening :)